packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -98,7 +122,7 @@ const EDiscovery = WebexPlugin.extend({
    * @param {number} options.timeoutMs - connection timeout in milliseconds, defaults to 30s
    * @returns {Promise<ResponseEntity<ReportRecord>>} Http response containing the specified report record
    */
-  getReport(reportId, options) {
+  async getReport(reportId, options) {
```

Comment:
Technically this is not needed. But I won't object to it either.

Reply:
Yeah, not needed atm but going for consistency across the APIs